stories/assembly-aap-gcp-extension-node.adoc

```diff
@@ -6,6 +6,6 @@ ifdef::context[:parent-context: {context}]
 :context: aap-gcp-nodes
 
 The extension node offering enables you to add additional managed node licenses and additional compute VMs into your {AAPonGCP} deployment. 
-You can deploy the multiple extension nodes if you desire more than one extra {ControllerName}.
+You can deploy multiple extension nodes if you desire more than one extra {ControllerName}.
```

Comment:
The PR deletes this assembly from stories.adoc. 
Should this assembly be deleted instead of stories/assembly-aws-extension-nodes.adoc ?

Reply:
Since I moved the extension nodes content from a separate section (previously section 8) to a subsection of 5, I removed the assembly.  I admit, this is my first time with asciidoc, so please let me know if something was done improperly.

Reply:
Spoke too soon: I see that you've restored the file.

Reply:
That was a mistake.  I realized that the file was for AWS.  I just restored.  Does it look ok now?

Reply:
Yes: the AWS doc is building OK, and the GCP build looks fine too.